flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -1839,6 +1839,28 @@ class FMCMainDisplay extends BaseAirliners {
     setApproachIndex(approachIndex, callback = EmptyCallback.Boolean) {
         this.ensureCurrentFlightPlanIsTemporary(() => {
             this.flightPlanManager.setApproachIndex(approachIndex, () => {
+                const frequency = this.flightPlanManager.getApproachNavFrequency();
+                if (isFinite(frequency)) {
+                    const freq = Math.round(frequency * 100) / 100;
+                    if (this.connectIlsFrequency(freq)) {
+                        this._ilsFrequencyPilotEntered = false;
+                        SimVar.SetSimVarValue("L:FLIGHTPLAN_APPROACH_ILS", "number", freq);
+                        const approach = this.flightPlanManager.getApproach();
+                        if (approach && approach.name && approach.name.indexOf("ILS") !== -1) {
+                            const runway = this.flightPlanManager.getApproachRunway();
+                            if (runway) {
+                                SimVar.SetSimVarValue("L:FLIGHTPLAN_APPROACH_COURSE", "number", runway.direction);
```

Comment:
Is this meant to be included in this PR ?

Reply:
Hi, no, good spot. This must've sneaked in on one of the merges, it looks like these lines were removed in PR #4973